srcpkgs/puyovs/template

```diff
@@ -0,0 +1,29 @@
+# Template file for 'puyovs'
+pkgname=puyovs
+version=2022.11.28
+revision=1
+build_wrksrc=${pkgname}-nightly
+build_style="cmake"
+short_desc="PuyoVS"
+maintainer="poopingpenis <poopingpeniz@outlook.com"
+license="GPL-3.0"
+homepage="https://puyovs.com/"
+distfiles="https://github.com/puyonexus/puyovs/archive/refs/tags/nightly.tar.gz https://github.com/g-truc/glm/archive/refs/tags/0.9.9.8.tar.gz>glm-v0.9.9.8.tar.gz https://github.com/zlib-ng/zlib-ng/archive/refs/tags/2.0.6.tar.gz>zlib-ng-v2.0.6.tar.gz https://github.com/randy408/libspng/archive/refs/tags/v0.7.2.tar.gz>libspng-v0.7.2.tar.gz https://github.com/libsdl-org/SDL_ttf/archive/refs/tags/release-2.20.1.tar.gz>SDL_ttf-v2.20.1.tar.gz https://github.com/libsdl-org/SDL/archive/refs/tags/release-2.24.1.tar.gz>SDL-v2.24.1.tar.gz https://github.com/libsdl-org/freetype/archive/6fc77cee03e078e97afcee0c0e06a2d3274b9a29.tar.gz>freetype-6fc77cee03e078e97afcee0c0e06a2d3274b9a29.tar.gz https://github.com/libsdl-org/harfbuzz/archive/6022fe2f68d028ee178284f297b3902ffdf65b03.tar.gz>harfbuzz-6022fe2f68d028ee178284f297b3902ffdf65b03.tar.gz"
```

Comment:
None of this should be using vendored copies ever, at all, for any reason. :)

Reply:
No vendored libraries, must be a versioned `distfile`.